tasty-plutus/src/Test/Tasty/Plutus/WithScript.hs

```diff
@@ -89,121 +80,18 @@ withValidator name val (WithSpending comp) =
 
  = Important note
 
- Unless your 'MintingPolicy' has been prepared using 'toTestMintingPolicy',
- this will likely not behave as intended.
+ Unless your 'TestMintingPolicy' has been prepared using 'toTestMintingPolicy',
+ this will likely not behave as intended. We use 'WrappedMintingPolicy' to prevent
+ other wrapper functions from being used.
 
  @since 3.0
```

Comment:
Due to the API change on this function, the `since` should be updated to match the version in which it changed.